### ISSUE TITLE
away.log should clear when the user returns from away status

### ISSUE THREAD

**Issue by dkg** [irssi/irssi#810]
I was surprised to find messages dating back years in ~/.irssi/away.log.  I haven't been `/away` for years! :)

when the user returns from `/away`, the away log should be truncated.

**Comment by ailin-nemui** [comment]
currently Irssi is designed to keep the awaylog, just like your other logs. You can use cleaning or rotating scripts

**Comment by dequis** [comment]
Yeah not a fan of this idea, especially given how there's the user expectation that the file won't be cleared. It would be super rude to start truncating a file that was previously not truncated for years.

**Comment by lelutin** [comment]
@ailin-nemui I'm wondering what use this has. I'm especially concerned by this default policy: can there be away messages from channels that specifically do no log that get kept indefinitely by this?

**Comment by KwadroNaut** [comment]
As a simple user, I expect that after returning, the away log gets displayed and flushed right then. I understand that a change would/could be seen as rude, but the default of keeping logs isn't quite nice either. The suggestion from @ailin-nemui in https://github.com/irssi/irssi/pull/789#issuecomment-366232058 could be a compromise where current behavior isn't improved/doesn't annoy anyone but changes it for new users?

**Comment by bmeh** [comment]
Perhaps could add a new setting that would do just that (upon coming back from away, print and clear), but default to the current behavior.

**Comment by duckdalbe** [comment]
I stumbled across this behaviour, too. My expectation was that away logs would be cleared once I'm back. I don't see any use for the older entries once they have been shown to me.

I can imagine that many long-term users learned about this behaviour and grew used to it, but I'd also suspect that the majority of new users do not expect this behaviour.

As a workaround I now `/set awaylog_file ~/.irssi/away-%u.log`, which reduces the data rentention time to one week. But I'd prefer a configuration option that makes irssi properly truncate the logfile
